projects/core/src/user/facade/user-consent.service.ts

```diff
@@ -119,7 +127,12 @@ export class UserConsentService {
    */
   getConsent(templateId: string): Observable<Consent | undefined> {
     return this.authService.isUserLoggedIn().pipe(
-      filter(Boolean),
+      distinctUntilChanged(),
```

Comment:
nitpick: Do we really need this `distinctUntilChanged`? I can see we already have it inside the implementation of `isUserLoggedIn`.

Reply:
This code has been tested also against `AsmAuthService` where `distinctUntilChanged` is missing, that's why I decided to leave it here. But if we decide to modify `AsmAuthService` and `AuthService` then of course, I prefer to have it there 👌